adagucserverEC/CImgRenderPolylines.cpp

```diff
@@ -229,7 +300,12 @@ void CImgRenderPolylines::render(CImageWarper *imageWarper, CDataSource *dataSou
 #ifdef MEASURETIME
         StopWatch_Stop("Feature drawn %d", featureIndex);
 #endif
-        featureIndex++;
+      }
+      CDBDebug("Drawing %d rects", rects.size());
```

Comment:
Remove or make it depend on a macro setting

Reply:
Done